modules/issue_tracker/test/issue_tracker_test_plan.md

```diff
@@ -1,37 +1,36 @@
-Issue Tracker Filter Form [Automation Testing]
-1. A user can access the module only if they have one or more of the `issue_tracker_reporter` or `issue_tracker_developer` permissions.
+# Issue Tracker Test Plan
+
+## Issue Tracker Filter Form [Automation Testing]
+1. User can access the page iff they have issue tracker reporter or developer permission
 2. User can see data from other sites iff they have access_all_profiles permission
 3. Test that all filters work. Nothing should be filtered at first loading.
-4. Test that all tabs (with filters) work and redirect to the correct table. My issues should be missing assignee filter and closed issues should be missing status filter. 
+4. Test that all tabs (with filters) work and redirect to the correct table. 
 5. Test that the watching checkbox works correctly (issues that your userID is watching in issues_watching table)
 6. Check that links to issues in table are correct.
 7. Check that table sorts and displays additional pages correctly 
 
-Issue Tracker Create New Issue [Manual Testing]
-1. User can access the page if they have reporter permission
+## Issue Tracker Create New Issue [Manual Testing]
+1. User can access the page if they have reporter or developer permission.
```

Comment:
this should have the permission name. 'reporter or developer permission' isn't clear.